pom.xml

```diff
@@ -65,7 +65,7 @@
   </distributionManagement>
 
   <properties>
-    <mavenVersion>3.1.0</mavenVersion>
+    <mavenVersion>3.1.1</mavenVersion>
```

Comment:
Please add xml comment: required do to conflict with asm:asm:3.3.1 in Maven 3.1.0

Reply:
Done, also added the maven issue number so others can verify it.